docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -423,6 +422,36 @@ import { Steps } from '@astrojs/starlight/components';
 
 </Steps>
 
+### Badges
+
+import { Badge } from '@astrojs/starlight/components';
+
+Utilisez le composant `<Badge>` pour afficher de petits éléments d'information, tels que des statuts ou des étiquettes.
+
+Passez le contenu que vous souhaitez afficher à l'attribut `text` du composant `<Badge>`.
+
+Par défaut, le badge utilise la couleur d'accentuation du thème de votre site. Pour utiliser une couleur de badge intégrée, définissez l'attribut `variant` à l'une des valeurs suivantes : `note` (bleu), `tip` (violet), `danger` (rouge), `caution` (orange), ou `success` (vert).
+
+L'attribut `size` (par défaut : `small`) contrôle la taille du texte du badge. Les attributs `medium` et `large` sont également disponibles pour afficher un badge plus grand.
```

Comment:
```suggestion
L'attribut `size` (par défaut : `small`) contrôle la taille du texte du badge. Les valeurs `medium` et `large` sont également disponibles pour afficher un badge plus grand.
```

To make sure it's obvious these are not other props but valid values for `size`.